dff/config/superset_dashboard/dashboards/DFF_statistics_dashboard_1.yaml

```diff
@@ -0,0 +1,1315 @@
+        \ ratings from user requests as a numeric variable. Much like the previous\
+        \ example, you can employ a variety of charts to analyze this data, such as\
+        \ line charts or box plots. \n\nNote that to reproduce the chart below the\
+        \ value will have to be cast to Int16 or to some other numeric datatype to\
+        \ find the average.\n\n```sql\nAVG(CAST(JSON_VALUE(data, '$.key') AS Int16))\n\
+        ```"
+      height: 33
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-q4CdGUDlq
+    - ROW-Hcn9FCMH6a
+    type: MARKDOWN
+  MARKDOWN-PY70BXaRGq:
+    children: []
+    id: MARKDOWN-PY70BXaRGq
+    meta:
+      code: "# Overview\n\nThe `overview` section is a generalized representation\
+        \ of the data related to your service. It allows you to inspect raw data entries\
+        \ or to observe some general trends using weighted charts based on the dialog\
+        \ graph.\n\n## Raw data\n\nRaw data shows unaggregated data points as a table.\
+        \ \nThe `data` column offers a closer view of the collected keys and labels\
+        \ in the form of JSON objects.\n\n<br />\n"
+      height: 33
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-JCU6rANFP
+    - ROW-R1SOLtdPpz
+    type: MARKDOWN
+  MARKDOWN-XuW-kEED70:
+    children: []
+    id: MARKDOWN-XuW-kEED70
+    meta:
+      code: '# Node statistics
+
+
+        The `Node statistics` section mostly contains data related to how frequently
+        individual nodes are visited with the aim of providing you insights on how
+        well distinct parts of the graph function.
+
+
+        ## Node visits distribution
+
+
+        The charts below display the number of times that the flows and nodes of the
+        dialog graph were visited during graph traversal. These statistics are supposed
+        to help the developers better understand the importance of any particular
+        node inside the graph.'
+      height: 35
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-Gw0Ffh1lG
+    - ROW-t7PnH6oKHU
+    type: MARKDOWN
+  MARKDOWN-aXRTn3gE5x:
+    children: []
+    id: MARKDOWN-aXRTn3gE5x
+    meta:
+      code: '## Transitions distribution
+
+
+        The chart below shows, how frequently each of the graph edges was traversed
+        by various users.
+
+
+        Based on this information, you can determine which transitions inside the
+        dialog graph function
+
+        as intended. If some edge has never been traversed, it is safe to draw the
+        conclusion that transition
+
+        conditions need to be adjusted.'
+      height: 24
+      width: 12
+    parents:
+    - ROOT_ID
+    - GRID_ID
+    - TABS-Xoi5oUBxZI
+    - TAB-Gw0Ffh1lG
+    - ROW-y6AC1wLv1j
+    type: MARKDOWN
+  MARKDOWN-i9daMe46XY:
+    children: []
+    id: MARKDOWN-i9daMe46XY
+    meta:
+      code: '## Requests and responses
```

Comment:
Why do requests and responses have dates as `start_time` (without times)?
![image](https://github.com/deeppavlov/dialog_flow_framework/assets/61429541/f188b7fb-187f-4da1-b55b-888b0ff8efb8)